modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -190,8 +190,8 @@ The following settings are used to configure for this scenario:
 * Set `dbms.cluster.discovery.resolver_type=K8S`.
 * Set `xref:configuration/configuration-settings.adoc#config_dbms.kubernetes.label_selector[dbms.kubernetes.label_selector]` to the label selector for the cluster services.
 For more information, see the https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors[Kubernetes official documentation^].
-* Depending on your discovery service version, set either `xref:configuration/configuration-settings.adoc#config_dbms.kubernetes.service_port_name[dbms.kubernetes.service_port_name]` (v1), or xref:configuration/configuration-settings.adoc#config_dbms.kubernetes.discovery.v2.service_port_name[`dbms.kubernetes.discovery.v2.service_port_name]` (v2) to the name of the service port used in the Kubernetes service definition for the Core's discovery port.
-For more information, see the https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#serviceport-v1-core[Kubernetes official documentation^].
+* Set xref:configuration/configuration-settings.adoc#config_dbms.kubernetes.discovery.service_port_name[`dbms.kubernetes.discovery.service_port_name]` to the name of the service port used in the Kubernetes service definition for the Core's discovery port.
```

Comment:
```suggestion
* Depending on your discovery service version, set either `xref:configuration/configuration-settings.adoc#config_dbms.kubernetes.service_port_name[dbms.kubernetes.service_port_name]` (v1), or xref:configuration/configuration-settings.adoc#config_dbms.kubernetes.discovery.v2.service_port_name[`dbms.kubernetes.discovery.v2.service_port_name]` (v2) to the name of the service port used in the Kubernetes service definition for the Core's discovery port.
```

Reply:
After 5.23 (if I'm not mistaken), you can select a version of discovery service.